Sources/Examples/Echo/Generated/echo.grpc.swift

```diff
@@ -212,7 +212,11 @@ class Echo_EchoServiceTestStub: ServiceClientTestStubBase, Echo_EchoService {
     return getResponses.first!
   }
   func get(_ request: Echo_EchoRequest, metadata customMetadata: Metadata, completion: @escaping (Echo_EchoResponse?, CallResult) -> Void) throws -> Echo_EchoGetCall {
-    fatalError("not implemented")
+    let response = try self.get(request)
+    let callResult = CallResult(success: true, statusCode: .ok, statusMessage: "OK", resultData: nil,initialMetadata: nil, trailingMetadata: nil)
```

Comment:
nit: missing space before `initialMetadata` (also in the code generator)

Reply:
Sure

Reply:
Still missing a space before `initialMetadata`, also in the codegen